### ISSUE TITLE
Vete a tomar por culo

### ISSUE THREAD

**Issue by SrNemuri** [CodeCube-team/webpage#1]

**Comment by JuanjoTorres** [comment]
Yo quitaría los apartados _FAQ_, _Reviews_ _Screens_ y _Download_.

**_Screens_** es muy importante para nuestro _landing page_ porque demuestra a los futuros clientes las posibilidades que tienen a la hora de contratar nuestros servicios; pero como ahora mismo no tenemos ninguno es mejor cerrar ese apartado. **_FAQ_**, _**Reviews**_ esta claro, no hemos generado los clientes suficientes para tener activo estos apartados. Y **_Download_**, no tenemos ninguna versión para descargar, ni falta que nos hace ahora mismo.

Los demás apartados creo que son los que necesitamos fundamentalmente.

- _**Features**_, les explicamos que destacamos sobre las demás compañías.

- _**Team**_, nos presentamos a nosotros mismos con nuestros cargos, para simpatizar y crear un vínculo más cercano.

- _**Contact**_, para que se puedan poner en contacto con nosotros.

También sería interesante mirar de colocar imágenes acorde con lo que ofrecemos, en el caso de los smartphones. (Veo que este _template_ era solo para dispositivos móviles porque no veo ninguna imagen de como se vería en un _laptop_). Sobretodo personalizar las _URLs_ y el contenido de la página.

- [ ] Crear contenido relevante y las _URLs_.

- [ ] Buscar imágenes de _laptops_ sin imagen en su pantalla.

- [ ] Eliminar o comentar todo lo no necesario para el principio (cualquier botón de _Play Store_ o _Descarga_).

- [ ] Crear varios diseños para las imágenes de los dispositivos (_Adobe XD Beta_ estaría guay).

Otros detalles que no he dicho son: incluir _Google Analytics_, suprimir _Google Maps_ y dirección postal, quitar el recuento de _Seguidores_, _Descargas_ y _Reviews_, añadir la licencia (Copyright, Copyleft, CCommons...). Después de tener esto bien estabilizado, empezaría por crear un proyecto sencillo pero notorio, así que pensad en algo de mientras.

¿Qué os parece? @SrNemuri @ybustos @xixou1

**Comment by xixou1** [comment]
Yo el tema de FAQ quizás si que lo dejaba, donde podriamos poner las típicas preguntas fáciles de contestar del estilo: "¿Tipo de webs que proporcionamos ? , ¿Tecnologias que usamos ? ", etc.

En cuanto a los otros 2 que Juanjo ha mencionado, no tengo objeción ninguna, la pestaña de Pricing y Donwload las quitaba.

Quedarían así en total:

Features, Team. (Aqui añadiria la seccion de Screens en un futuro), Contact, y por ultimo FAQ.

**Comment by JuanjoTorres** [comment]
La sección Pricing como dices las quitaba ahora pero piensa que luego más adelante la necesitaremos para advertir o mejor dicho, informar a la gente interesada de nuestras tarifas o planes.

**Comment by SrNemuri** [comment]
Estoy de acuerdo en parte. La parte de **_Pricing_** nos vendrá bien en el futuro, pero solamente para mantenimiento y para apps que podamos llegar a sacar. Por ahora creo que los trabajos que nos pueden salir son de hacer webs a medida, por lo que no podemos ofrecer una tabla de precios. Por otra parte, nos interesa que el cliente se vea "obligado" a contactar con nosotros para hablar de precios, así lo podremos "enganchar"
